src/api/rest.ts

```diff
@@ -30,4 +30,9 @@ export const put = async (url: string, body: JSONValue, headers: Header[] = [])
   return axios.put(url, body, {
     headers: restructureHeaders(headers),
   });
+}
+export const delete_req = async (url: string, headers: Header[] = []) => {
```

Comment:
Shall we make the method name to `delete` to make it consistent like other methods get, post, patch, put etc!

Reply:
"delete" is a reserved key word , and cannot be used .